scripts/game_structure/windows.py

```diff
@@ -621,6 +621,83 @@ def process_event(self, event):
                 game.all_screens['profile screen'].screen_switches()
                 self.kill()
 
+class ExileCat(UIWindow):
```

Comment:
Was this intended to be added in the PR? It is not mentioned in the title or summary. It is also causing the pylint test fail.

Reply:
Yeah I'm just not sure how to create separate PRs ;-;

Reply:
I'd be happy to help! You need to create a new branch on your fork, and then you can put whatever changes you want on that branch, then PR from that branch.  This branch has a lot of stuff on it, so I suggest creating a new branch and copy-and-pasting all the changed you want for a single PR. Then creating a new PR from that fork.

Reply:
If you want a more step-by-step guide, message me on discord!